Mathlib/Combinatorics/SimpleGraph/Matching.lean

```diff
@@ -396,4 +407,18 @@ lemma IsPerfectMatching.symmDiff_spanningCoe_of_isAlternating {M : Subgraph G}
       simp_all only [show M.Adj v y ↔ ¬M.Adj v w' from by simpa using hG' hw'.1 hr.1 hw'.2, not_not,
         ne_eq, and_false]
 
+lemma IsPerfectMatching.symmDiff_spanningCoe_IsPerfectMatching_IsAlternating_left {M' : Subgraph G'}
+    (hM : M.IsPerfectMatching) (hM' : M'.IsPerfectMatching) :
+    (M.spanningCoe ∆ M'.spanningCoe).IsAlternating M.spanningCoe := by
+  intro v w w' hww' hvw hvw'
+  obtain ⟨v1, ⟨hm1, hv1⟩⟩ := hM.1 (hM.2 v)
+  obtain ⟨v2, ⟨hm2, hv2⟩⟩ := hM'.1 (hM'.2 v)
+  simp only [symmDiff_def] at *
+  aesop
+
+lemma IsPerfectMatching.symmDiff_spanningCoe_IsPerfectMatching_IsAlternating_right
```

Comment:
What about
```suggestion
lemma IsPerfectMatching.isAlternating_symmDiff_right
```
... taking the point of view that `spanningCoe` is a coercion

Reply:
I adapted this change to a couple other lemmas as well, feel free to let me know if I was overeager in that:)